README.md

```diff
@@ -50,6 +50,7 @@ Additionally, DNS-collector also support
     - [`ElasticSearch`](doc/loggers.md#elasticsearch-client)
     - [`Scalyr`](doc/loggers.md#scalyr-client)
     - [`Redis`](doc/loggers.md#redispub)
+    - [`Falco`](doc/loggers.md#falco)
```

Comment:
Can you add this Falco logger in a sub item (Security tools ?)

Reply:
Updated.